color.js

```diff
@@ -12,11 +12,23 @@ export class Color {
   }
 
   //"Factory method" that creates color from given HEX value
-  static fromHex(hexColor) {
+  static hexToRGB(hexColor) {
```

Comment:
I'm not sure if we should change names here because our color is clearly always stored in RGB format